contracts/validator-manager/ValidatorManager.sol

```diff
@@ -542,6 +545,7 @@ abstract contract ValidatorManager is Initializable, ContextUpgradeable, IValida
         uint64 newValidatorWeight,
         uint64 oldValidatorWeight
     ) private {
+        
```

Comment:
nit: extra lines and similar at other places. This is repo seems to have solhint, did you try applying that maybe?

Reply:
I didnt use linting, will check that